### PR TITLE
Update seed IPs, based on bitcoin.sipa.be crawler data

### PR DESCRIPTION
**Pull request by sipa**

### CODE REVIEW

**Comment by sipa** [comment]
Significant update:
* Updated the contrib/seeds/makeseeds.py script
* Include @gmaxwell's ASN filtering
* Updated the seeds from my crawler
* Sort the results for determinism and future comparability

**Comment by gavinandresen** [comment]
ACK

**Comment by gmaxwell** [comment]
ACK

File: contrib/seeds/makeseeds.py [hunk 1]
```diff
@@ -1,32 +1,118 @@
...
+
+def parseline(line):
+    sline = line.split()
+    if len(sline) < 11:
+       return None
+    # Match only IPv4
+    m = PATTERN_IPV4.match(sline[0])
+    if m is None:
+        return None
+    # Do IPv4 sanity check
```
Comment (pstratem) [inline contrib/seeds/makeseeds.py]: why not use socket.inet_aton ?

**Comment by pstratem** [comment]
ACK (beyond above bikeshedding)

File: contrib/seeds/makeseeds.py (hunk 1 above)
Comment (sipa) [inline contrib/seeds/makeseeds.py]: Suggest code, and I'll change it. I don't care enough to spend more time to look up how that works.

**Comment by laanwj** [comment]
Tested re-ACK w/ wiping peers.dat and -dnsseed=0,

```
DNS seeding disabled
... (wait some time)
Adding fixed seed nodes as DNS doesn't seem to be available.
receive version message: /Satoshi:0.9.3/: version 70002, blocks=337739, us=213.93.107.3:34254, peer=1
... (started syncing at high speed)
```

**Pull request merged**